jobs/jobs.yml

```diff
@@ -224,7 +225,7 @@
           python-version: "{python-version}"
           clear: true
           nature: shell
-          command: !include-raw: scripts/volto-i18n.sh
+          command: !include-raw: scripts/volto-i18n-{branch}.sh
```

Comment:
I don't think this will work, we usually use that branch within the script and use `if/else` there 😄 

I can amend it for you 👍🏾